src/programs/mbta.py

```diff
@@ -71,55 +28,221 @@ def print_text(lines):
+            display_error(["ERROR : ", "Invalid API", "key"])
+            print(f"{e}, Invalid API key.")
+        display_error(["ERROR : ", "Unable to", "fetch data"])
+        print(f"{e}, Unable to fetch predictions data.")
+    else:
+        # Stringify the promise to data
+        data = response.json()
+        return data
+
+
+def mock_fetch_predictions_data():
+    return {
+        "data": [
+            {
+                "attributes": {
+                    "arrival_time": "2023-07-18T22:58:28-04:00",
+                    "departure_time": "2023-07-18T21:59:25-04:00",
+                    "direction_id": 0,
+                    "schedule_relationship": "ADDED",
+                    "status": "FIRE",
+                    "stop_sequence": 40,
+                },
+                "id": "prediction-ADDED-1581584541-70069-40",
+                "relationships": {
+                    "route": {"data": {"id": "Red", "type": "route"}},
+                    "stop": {"data": {"id": "70069", "type": "stop"}},
+                    "trip": {"data": {"id": "ADDED-1581584541", "type": "trip"}},
+                    "vehicle": {"data": {"id": "R-5477AEE4", "type": "vehicle"}},
+                },
+                "type": "prediction",
+            },
+            {
+                "attributes": {
+                    "arrival_time": "2023-07-18T22:05:57-04:00",
+                    "departure_time": "2023-07-18T22:06:54-04:00",
+                    "direction_id": 0,
+                    "schedule_relationship": None,
+                    "status": None,
+                    "stop_sequence": 40,
+                },
+                "id": "prediction-57599385-70069-40",
+                "relationships": {
+                    "route": {"data": {"id": "Red", "type": "route"}},
+                    "stop": {"data": {"id": "70069", "type": "stop"}},
+                    "trip": {"data": {"id": "57599385", "type": "trip"}},
+                    "vehicle": {"data": {"id": "R-5477AF12", "type": "vehicle"}},
+                },
+                "type": "prediction",
+            },
+        ],
+        "jsonapi": {"version": "1.0"},
+        "links": {
+            "first": "https://api-v3.mbta.com/predictions?filter[direction_id]=0&filter[stop]=place-cntsq&page[limit]=2&page[offset]=0",
+            "last": "https://api-v3.mbta.com/predictions?filter[direction_id]=0&filter[stop]=place-cntsq&page[limit]=2&page[offset]=6",
+            "next": "https://api-v3.mbta.com/predictions?filter[direction_id]=0&filter[stop]=place-cntsq&page[limit]=2&page[offset]=2",
+        },
+    }
+
+
+def fetch_vehicles_data(id: str):
+    try:
+        # Fetch
+        response = requests.get(
+            url=f"https://api-v3.mbta.com/vehicles/{id}",
+            headers=headers,
+            auth=None,
+        )
+    except Exception as e:
+        display_error(["ERROR : ", "Unable to", "fetch vehicle", "data"])
```

Comment:
Same capitalization comment as above
```suggestion
        display_error(["Error : ", "Unable to", "fetch vehicle", "data"])
```